app/controllers/site.rb

```diff
@@ -21,17 +21,26 @@ class Site < Base
 						end
 					end.compact
 
-					slim :index, locals: { sites: policy_scope(Tint::Site), git_providers: git_providers }
+					sites = policy_scope(Tint::Site)
+					repos = sites.map { |site| site.remote }
+
+					slim :index, locals: { sites: sites, repos: repos, git_providers: git_providers }
 				end
 			end
 
 			post "/" do
 				authorize Tint::Site, :create?
 
+				if params[:provider] == "github"
+					identity = DB[:identities][user_id: pundit_user[:user_id], provider: "github"]
```

Comment:
What do you think of creating an Identity class that would take this in? Then it could have a `token` method instead of accessing by key. We already use the token in another route in this controller so it could abstract that access for us.

We could also refactor auth to use an identity.